### PR TITLE
Extend DISCOVERY_TOPIC to include physical information [14006]

### PR DESCRIPTION
**Pull request by EduPonz**
This PR documents eProsima/Fast-DDS#2549

### CODE REVIEW

**Comment by EduPonz** [comment]
@richiprosima please test this

File: docs/conf.py [hunk 1]
```diff
@@ -317,9 +317,13 @@ def configure_doxyfile(
 extensions = [
     'breathe',
     'sphinxcontrib.plantuml',
-    'sphinx.ext.autodoc'  # Document Pydoc documentation from Python bindings.
+    'sphinx.ext.autodoc',  # Document Pydoc documentation from Python bindings.
+    'sphinx_tabs.tabs'
 ]
 
+sphinx_tabs_disable_css_loading = False
```
Comment (JLBuenoLopez-eProsima) [inline docs/conf.py:324]: Could it be that this option is the one disabling the loading of the tabs css?

File: docs/fastdds/property_policies/non_consolidated_qos.rst [hunk 2]
```diff
@@ -117,6 +118,96 @@ In this case, the property value is a semicolon separated list containing the
 |    :end-before: <!--><-->                                                                                            |
 +----------------------------------------------------------------------------------------------------------------------+
 
+.. _property_policies_physical_data:
+
+Physical Data in Discovery Information
+""""""""""""""""""""""""""""""""""""""
+
+It is possible to include the information conveyed in the :ref:`statistics_topic_names_physical` into the participant
+discovery message, a.k.a *DATA[p]*.
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/property_policies/non_consolidated_qos.rst:127]: Mayne a link to the discovery phases section could help the reader.

File: docs/fastdds/property_policies/non_consolidated_qos.rst [hunk 3]
```diff
@@ -117,6 +118,96 @@ In this case, the property value is a semicolon separated list containing the
...
+     - Not set
+     - ``""``
+   * - ``"fastdds.physical_data.process"``
+     - Name of the process running the application
+     - Not set
+     - ``""``
+
+Whenever any of these properties is defined within the |DomainParticipantQos-api|, the |DomainParticipant-api| *DATA[p]*
+will contained the set value.
+Furthermore, if any of these properties is set to a value of ``""`` (the default when ``FASTDDS_STATISTICS`` is
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/property_policies/non_consolidated_qos.rst:153]: Maybe a link to the CMake options section might help the reader understand what you are referring to.

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
Only a couple of nitpicks

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Pull request merged**